### DISCUSSION TITLE
如何安装office365版本

### DISCUSSION THREAD

**Discussion by AltimLi** [OdysseusYuan/Mocreak#24] [General]
安装界面看不到任何安装office365，是还不支持吗。

**Comment by OdysseusYuan** [comment]
感谢您的关注。

目前版本尚未支持 Office 365 部署，但已支持协同办公。

您是否额外需要 Office 365 的“云服务”模块？

**Reply by AltimLi** [reply]
不需要云服务

**Reply by OdysseusYuan** [reply]
了解。如果不需要云服务，2024 版本可以满足您的日常功能性使用？
